login/login-page.xml

```diff
@@ -2,34 +2,36 @@
     <ActionBar title="Login" class="action-bar">
     </ActionBar>
 
-    <StackLayout class="p-t-15">
-        <Button class="btn btn-outline text-left" tap="onLoginFacebookButtonTap">
-            <FormattedString>
-                <!-- 
-                    To see icons, you need to add font awesome to your project.
-                    Learn more about icon font and how it can be added in this documentation article:
-                    https://docs.nativescript.org/ui/icon-fonts
-                -->
-                <Span text="    &#xf09a;" class="fa"></Span>
-                <Span text="     Log in with Facebook"></Span>
-            </FormattedString>
-        </Button>
+    <ScrollView orientation="vertical">
```

Comment:
`vertical` is the default ScrolLView orientation?

Reply:
👍  I will remove it :)